src/chronos/wTraveler/update-user-handler.ts

```diff
@@ -15,7 +16,41 @@ export function handleUpdateUser(event: UpdateUser): void {
   const user = event.params.user.toHexString();
   const expires = event.params.expires;
 
+  if (expires.lt(event.block.timestamp)) {
+    log.warning(
+      "[handleUpdateUser] Expiration date is in the past, ignoring event, tx: {}",
+      [event.transaction.hash.toHex()]
+    );
+    return;
+  }
```

Comment:
Even if the new `expires` is lower than the current one and does not create a new rental, it will still end the previous one **if the previous one is not expired yet**.